packages/vite/src/node/cli.ts

```diff
@@ -235,6 +252,6 @@ cli
   )
 
 cli.help()
-cli.version(require('../../package.json').version)
+cli.version(require('vite/package.json').version)
```

Comment:
**question:** Why is this needed, I don't see the correlation to the PR

Reply:
no it's unrelated, because during I fix the CI problem, the debug info refer to `package.json` something, so I change to this and try to see if it cause the problem. The turn out show it has nothing to do with the problem

but they are totally equal, just using the alias, you can also find here 
https://github.com/vitejs/vite/blob/828a8e538ce289922b9301339be4d5d6ea84b496/packages/vite/src/node/server/index.ts#L602

Reply:
I think we should revert this change as this is unrelated to the PR as you stated.